tests/test_utils/alloc.py

```diff
@@ -100,5 +103,7 @@ def run_it(f):
 
     if gc_was_enabled:
         gc.enable()
-        # forcibly clean up anything left over in this benchmark
-        gc.collect()
+
+    # This benchmark is finished, and now forcibly clean up any garbage left over from the
+    # small/partial collections that were being during the benchmark.
```

Comment:
My impression was that the final `gc.collect()` now cleans up the long-term objects created during the benchmark - or have I got that wrong?

Reply:
Yes, that's right; the benchmark does a few small collections, so the long term objects crated by each call to the benchmarking function may not have been collected. This call then collects those (if they're collectable at all). I guess the comment is a bit unclear, potentially because it's riddled with typos... let me fix those.